sphinx/source/docs/user_guide/data.rst

```diff
@@ -0,0 +1,240 @@
+.. _userguide_data:
+
+Sharing Data Between Plots
+==========================
```

Comment:
Maybe suggest: "Providing Data for Plots and Tables" ? I think a full chapter on CDS in and data representation in general is welcome and long over due, but I'd consider sharing between plots one specific sub-case of that